compiler/rustc_target/src/spec/mipsel_unknown_none.rs

```diff
@@ -0,0 +1,32 @@
+// Bare MIPS32r2, little endian, softfloat
+// can be used for MIPS M4K core (e.g. on PIC32MX devices)
+
+use crate::spec::{LinkerFlavor, LldFlavor, RelocModel};
+use crate::spec::{PanicStrategy, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "mipsel-unknown-none".to_string(),
+        target_endian: "little".to_string(),
+        pointer_width: 32,
+        target_c_int_width: "32".to_string(),
+        data_layout: "e-m:m-p:32:32-i8:8:32-i16:16:32-i64:64-n32-S64".to_string(),
+        arch: "mips".to_string(),
+        target_os: "none".to_string(),
+        target_env: String::new(),
+        target_vendor: String::new(),
+        linker_flavor: LinkerFlavor::Lld(LldFlavor::Ld),
+
+        options: TargetOptions {
+            cpu: "mips32r2".to_string(),
+            features: "+mips32r2,+soft-float,+noabicalls".to_string(),
```

Comment:
I don't think you have to turn on `+mips32r2` again, since it's already the `cpu`, right?

Reply:
No sure. The Linux targets specify mips32r2 in both the cpu as well as in the features. I just took this from the Linux target specs (BTW, I did not find a special feature flag for e.g. the M4K core in llvm/clang).

Reply:
Okay, seems fine to leave it in then.